.github/CODEOWNERS

```diff
@@ -621,3 +621,5 @@
 /singlestoredb_cloud/*metadata.csv     @tdasarathan @DataDog/ecosystems-review
 /singlestoredb_cloud/manifest.json     @tdasarathan @DataDog/ecosystems-review
 /singlestoredb_cloud/README.md         @tdasarathan @DataDog/ecosystems-review
+/dylibso-webassembly/manifest.json		@nilslice @bhelx @DataDog/ecosystems-review
+/dylibso-webassembly/README.md			@nilslice @bhelx @DataDog/ecosystems-review
```

Comment:
It looks like this is still failing a validation. we are looking at this in the meantime but could you try adding the metadata file here as well like singlestore did? I believe the validation is looking for all three files here.

Reply:
@emarsha94 - sure thing, just added along with an empty `metadata.csv`.